src/Resources/config/services.yml

```diff
@@ -19,3 +19,17 @@ services:
         arguments:
             - "@solutiondrive.factory.product_bundle"
             - "@solutiondrive.factory.product_bundle_slot"
+
+    solutiondrive.product_bundles.fixture.example_factory.product_bundle:
```

Comment:
Should the example-factories be defined in the 'main'-services.yml? Maybe you don't really want these services in your prod-environment...

Reply:
Neither sylius nor any plugin I've seen makes a difference between prod and dev-services. So I don't think we should..

Reply:
ok, was only a thought... :-)